docs/dns.md

```diff
@@ -1,12 +1,11 @@
 # DNS Support
-If you point a domain or subdomain to Google Cloud DNS, you can use `kctf-setup-dns` and `kctf-chal-dns` to map *[challenge_name].yourdomain.com* to your challenge.
+If you point a domain or subdomain to Google Cloud DNS as an NS record, kCTF creates a DNS zone that maps the challenge to your challenge's IP address.
 
 Follow these steps:
 
-1. Configure the subdomain in `kctf-setup-config-create`. 
+1. Configure the domain in `kctf-setup-config-create --domain-name foo.example.com`. 
 
-   Example: If you enter *foo.example.com*, the challenges will be deployed on *[challenge-name].foo.example.com*.
-1. Run `kctf-setup-dns` to create the DNS zone in Google Cloud DNS.
+   Example: If you enter *foo.example.com*, the challenges are be deployed on *xxxxx.foo.example.com*.
+1. Run `kctf-cluster-start`
+1. Add `DNS=true` on `config/challenge.conf`.
```

Comment:
this is now in challenge.yaml

Reply:
this merges to alpha, though.

Reply:
ah thats what you mean..

Reply:
lets fix this for alpha, we can fix beta independently